src/graphql/models/Badge.js

```diff
@@ -0,0 +1,16 @@
+import { GraphQLObjectType, GraphQLString } from 'graphql';
+
+export default new GraphQLObjectType({
+  name: 'Badge',
+  fields: () => ({
+    name: { type: GraphQLString },
+    displayName: { type: GraphQLString },
+    description: { type: GraphQLString },
+    link: { type: GraphQLString },
+    icon: { type: GraphQLString },
+    borderImage: { type: GraphQLString },
+    issuers: { type: GraphQLString },
+    createdAt: { type: GraphQLString },
+    updatedAt: { type: GraphQLString },
```

Comment:
Given the fact that some of these fields are required (not optional) in [schema](https://github.com/cofacts/rumors-db/blob/master/schema/badges.ts#L11-L22), we can safely declare that these fields are non-null.

Also, `issuers` should be a list of strings rather than a string. I am also not sure if we should reveal `issuers` field in this public-facing API.

This help reduce unnecessary null handling in API clients like rumors-site.

```suggestion
    name: { type: new GraphQLNonNull(GraphQLString) },
    displayName: { type: new GraphQLNonNull(GraphQLString) },
    description: { type: new GraphQLNonNull(GraphQLString) },
    link: { type: new GraphQLNonNull(GraphQLString) },
    icon: { type: new GraphQLNonNull(GraphQLString) },
    borderImage: { type: new GraphQLNonNull(GraphQLString) },
    issuers: { type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(GraphQLString))) },
    createdAt: { type: new GraphQLNonNull(GraphQLString) },
    updatedAt: { type: GraphQLString },
```